paladin-quests.lic

```diff
@@ -75,9 +87,54 @@ class PaladinQuests
     echo '***CONGRATS YOU NOW KNOW THE GLYPH OF WARDING ABILITY***'
   end
 
-  def bonding
-    echo '***UNSUPPORTED AT THIS TIME. PLEASE CONSIDER MAKING AN ISSUE AND' \
-         ' PROVIDING THE LOGS FOR ADDITION***'
+  def bonding(leader, room)
+    return unless leader
+    return unless room
+
+    echo "***THE BONDING QUEST IS STILL A WORK IN PROGRESS. THIS IS MORE OF AN ESCORT/HAND HOLDING THAN AUTO COMPLETE (AT THIS TIME)***"
+    echo "***YOU WILL BE EXPECTED TO DO THINGS THROUGHOUT, PREPARE YOURSELF***"
+
+    walk_to(room)
+
+    case bput("ask #{leader} about glyph", 'learn the Glyph of Bonding', 'the Citizenship Office')
+    when 'the Citizenship Office'
+      echo '***YOU DO NOT MEET THE REQUIREMENTS***'
+      echo "***PLEASE SUPPLY THIS LOG AS AN ISSUE IF YOU GET A 'NO MATCH FOUND' MESSAGE***"
+      exit
+    end
+
+    echo "***PAUSING SCRIPT. FIND THE ROOM DISPLAYED IN THE VISION. IT WILL BE IN THIS TOWN! UNPAUSE WHEN YOU FIND IT!***"
```

Comment:
What do you think about saying _how_ to unpause? I don't know if that's something most people are familiar with (`... ;unpause paladin-quests ...`)

Reply:
Actually meant to add this, thanks for the reminder!